### PR TITLE
migrate reagent part 3

### PR DESCRIPTION
**Pull request by flexsurfer**
migrate reagent part 3

### CODE REVIEW

File: src/quo/components/header.cljs [hunk 1]
```diff
@@ -133,76 +132,77 @@
...
+           background]
+    :or   {title-align   :center
+           border-bottom false}}]
+  (let [theme               (quo.theme/use-theme-value)
+        [layout set-layout] (rn/use-state {:left  {:width  (or left-width 8)
+                                                   :height header-height}
+                                           :right {:width  (or right-width 8)
+                                                   :height header-height}
+                                           :title {:width  0
+                                                   :height header-height}})
```
Comment (clauxx) [inline src/quo/components/header.cljs:148]: Might be better to split this into 3 separate states.

File: src/quo/components/tabs/tabs/view.cljs [hunk 2]
```diff
@@ -118,87 +116,81 @@
   - `scroll-on-press?` When non-nil, clicking on a tag centers it the middle
     (with animation enabled).
   "
-  [{:keys [default-active fade-end-percentage]
```
Comment (clauxx) [inline src/quo/components/tabs/tabs/view.cljs:121]: There's [this PR](https://github.com/status-im/status-mobile/pull/19033) from @ajayesivan that handles migrating this component as well

File: src/quo/components/header.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/quo/components/header.cljs:148]: to be honest i don't want to change logic much just migrate away from reagent

File: src/quo/components/tabs/tabs/view.cljs (hunk 2 above)
Comment (flexsurfer) [reply src/quo/components/tabs/tabs/view.cljs:121]: yeah that's fine

**Review by clauxx — APPROVED** [review]

**Comment by J-Son89** [comment]
Hard to know if these pr's had QA or not, but if not why is that?

**Pull request merged**